src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -442,6 +445,9 @@ def energyConsumption(dict_values, group):
             )
 
         if FILENAME in dict_values[group][asset]:
+            dict_values[group][asset].update(
+                {DISPATCHABILITY: {VALUE: False, UNIT: TYPE_BOOL}}
+            )
```

Comment:
I think this is missing from the `Changelog.md`? "Changed: The `DISPATCHABILITY` of energyConsumption assets is now set to FALSE by default".